js/main.js

```diff
@@ -1,35 +1,122 @@
 
-// open land acknowledgement via button
-openLaBtn.addEventListener("click", () => {
-	landAckModal.classList.remove("hidden");
-});
+// Team Modals Data
+
+let teamBioData = {
+	natasha: {
+		imgPath: "/assets/images/pages/team/natasha-rabsatt.png",
+		name: "Natasha Rabsatt, M.S.",
+		role: "Co-Founder",
+		email: "info@iftheselandscouldtalk.org",
+		linkedin: "https://www.linkedin.com/in/natasha-rabsatt/",
+		bio: [
+			"Natasha is Caribbean afro-indigenous with Classic Taíno (Puerto Rico - Boríken) and African heritage.",
+			"She is passionate about creating value and social impact through mindful stewardship with innovation, technology, and creative experiences. She is an avid world traveler and has lived in many different regions of the United States and also abroad.",
+			"Professionally, Natasha is a strategic designer, tech strategist, advisor, U.S. Army veteran, and entrepreneur. She has nearly 2 decades of experience in diverse industries and sectors leading people and directing initiatives and programs centered around learning, design, innovation, business development, and technology. Over the years, she has supported a number of businesses and organizations through the design and development of inclusive learning experiences, business development, innovation strategy, creative design, and Immersive Technology - AR/VR, Emerging Technologies - NFTs, Blockchain, Web 3.0 initiatives, and more.",
+		],
+	},
+	marsha: {
+		imgPath: "/assets/images/pages/team/marsha-maxwell.png",
+		name: "Dr. Marsha Maxwell, CETL ",
+		role: "Co-founder",
+		email: "info@iftheselandscouldtalk.org",
+		linkedin: "https://www.linkedin.com/in/natasha-rabsatt/",
+		bio: [
+			"With strategic, tactical and operational technology experience, Dr. Marsha Maxwell creates innovative environments and applications that harness the power of emerging technologies and neuroscience to enhance the educator's reach to better engage students and foster a culture of innovation in organizations.",
+			"Her areas of interest include Augmented Analytics, Immersive Experiences, Artificial Intelligence, Machine Learning, STEAM programs, and cybersecurity. Marsha has also been very active in bringing STEM/STEAM initiatives to life especially for underserved and underrepresented groups throughout their educational journey from kindergarten to university and beyond.",
+			"Marsha has been a featured speaker for various organizations' educational summits and workshops including Microsoft, Apple, and Google for her skills in education technology, innovation leadership, strategic analysis and decision-making, and equity and inclusion in technology.",
+		],
+	},
+};
+
+// Team Modal Functions
+
+// populate modal
+const populateTeamModal = profile => {
+	// deconstructing object
+	const { imgPath, name, role, bio, email, linkedin } = profile;
+
+	// Team modal elements
+	let memberImg = document.querySelector("#member-img");
```

Comment:
Nitpick: these could be `const` instead of `let`

Reply:
Sounds good!